.github/ionic-issue-bot.yml

```diff
@@ -24,12 +24,22 @@ closeAndLock:
   lock: true
   dryRun: false
 
+noReply:
```

Comment:
So what exactly is this block doing? I can gather a lot from the config. But does this help inform the other tasks that will do things like closing after x days?

Reply:
When a comment is received on an issue, if the issue has the `label: Awaiting Reply`, we remove said label and apply the `responseLabel: Reply Received` (because we set `dryRun: false`, the changes in labels will actually happen).

We set `lock: false` and `close: false` as a safety precaution - this action shouldn't close and/or lock an issue, because that'd be super frustrating. Imagine replying to the Stencil team (probably after doing some work to help us understand an issue), only to get your issue tagged with 'Reply Received' **and** then locked & closed.

Reply:
I suggest adding a "days" and a "message" to maybe give add a reminder message after X days without a reply (let's say 30).

If you leave "close" and "lock" false, this should just ping them a message in the sweep without swapping the label, but swap the label if someone else replies.

Reply:
> We set lock: false and close: false as a safety precaution - this action shouldn't close and/or lock an issue, because that'd be super frustrating. Imagine replying to the Stencil team (probably after doing some work to help us understand an issue), only to get your issue tagged with 'Reply Received' and then locked & closed.

In reality, it should only do a close or lock if you have `days` set, but since I suggest having `days` set with a `message`, then if you do not want the `close` and `lock` then yeah, have those `false`.

FWIW, I think the Framework at least does the `close` if no one responds. A `close` without a `lock` with a message like the FW uses may be appropriate as well. 🤷‍♂️

Reply:
The FW only gives ppl 14 days to respond. Seems short to me, but hey... They also close AND lock. I would probably just to with close and no lock to give the person a chance to respond if they forgot about it or whatever, and then let the "stale" task do the lock after a much longer period.

https://github.com/ionic-team/ionic-framework/blob/main/.github/ionic-issue-bot.yml#L100-L116